kompose.spec

```diff
@@ -190,11 +185,15 @@ export GOPATH=$(pwd):%{gopath}
 export GOPATH=$(pwd):$(pwd)/vendor:%{gopath}
```

Comment:
I'm not sure I quite understand adding `$(pwd)/vendor` to `$GOPATH`. It won't actually work because the vendor directory does not have a `src` subdir which is what `GOPATH` expects.

GOPATH expects:

```
$ tree -L 1 $GOPATH
/sharedfolder/code/go/
├── bin
├── pkg
└── src
```

where vendor has something like: 

```
$ tree -L 1 vendor/
vendor/
├── github.com
├── golang.org
├── google.golang.org
├── gopkg.in
└── k8s.io
```